app_dart/test/service/scheduler_test.dart

```diff
@@ -342,6 +342,25 @@ targets:
             throwsA(predicate((Exception e) => e.toString().contains('$branch is not enabled'))));
       });
 
+      test('checks for release branch regex', () async {
```

Comment:
Should this also be added to the integration tests? free form regular expressions are error prone.

Reply:
I see, so we have to land the engine PR first?

Reply:
Yes